[MinGW / Clang] LARGE_INTERGER instantiation warning

Hey guys,

Just noticed 2 small warnings in my Jenkins build process from `sentry-native`. Here is the output :

```
[283/363] Building CXX object sentry-native/CMakeFiles/sentry.dir/src/backends/sentry_backend_crashpad.cpp.obj
In file included from ../sentry-native/src/backends/sentry_backend_crashpad.cpp:5:
In file included from ../sentry-native/src/sentry_backend.h:6:
In file included from ../sentry-native/src/sentry_scope.h:6:
In file included from ../sentry-native/src/sentry_session.h:7:
../sentry-native/src/sentry_utils.h:130:46: warning: missing field 'HighPart' initializer [-Wmissing-field-initializers]
    static LARGE_INTEGER qpc_frequency = { 0 };
                                             ^
../sentry-native/src/sentry_utils.h:130:44: warning: suggest braces around initialization of subobject [-Wmissing-braces]
    static LARGE_INTEGER qpc_frequency = { 0 };
                                           ^
                                           {}
2 warnings generated.
```

Something like `static LARGE_INTEGER qpc_frequency = { {0,0} };` do silence them both.

--- a/src/sentry_utils.h
+++ b/src/sentry_utils.h
@@ -127,7 +127,7 @@ static inline uint64_t
 sentry__monotonic_time(void)
 {
 #ifdef SENTRY_PLATFORM_WINDOWS
-    static LARGE_INTEGER qpc_frequency = { 0 };
+    static LARGE_INTEGER qpc_frequency = { {0,0} };
 
     if (!qpc_frequency.QuadPart) {
         QueryPerformanceFrequency(&qpc_frequency);

--- a/src/sentry_utils.h
+++ b/src/sentry_utils.h
@@ -127,7 +127,7 @@ static inline uint64_t
 sentry__monotonic_time(void)
 {
 #ifdef SENTRY_PLATFORM_WINDOWS
-    static LARGE_INTEGER qpc_frequency = { {0,0} };
+    static LARGE_INTEGER qpc_frequency = { { 0, 0 } };
 
     if (!qpc_frequency.QuadPart) {
         QueryPerformanceFrequency(&qpc_frequency);

